cluster/addons/dns/kube2sky/RELEASES.md

```diff
@@ -1,43 +0,0 @@
-# Cutting a release
-
-Until we have a proper setup for building this automatically with every binary
-release, here are the steps for making a release. We make releases when they
-are ready, not on every PR.
-
-1. Build the container for testing: `make container PREFIX=<your-docker-hub> TAG=rc`
```

Comment:
I think these release steps are still valid for kubedns, we just need to modify to say something like: first build the binary with ./hack/build-go, then cd build/kube-dns and run make etc.